rasa_core/policies/keras_policy.py

```diff
@@ -56,9 +62,10 @@ def __init__(self,
         self.current_epoch = current_epoch
 
     def _load_params(self, **kwargs: Dict[Text, Any]) -> None:
-        config = copy.deepcopy(self.defaults)
+        config = copy.deepcopy(dict(self.defaults, **self.tf_defaults))
```

Comment:
we don't need self.tf_defaults anymore, because if there is no tf_config, we init tf.Session with None

Reply:
if I delete `self.tf_defaults` in `keras_policy.py` and `embedding_policy.py`, then there is no need have property tf_defatults in `policy.py`

Reply:
And I delete all tf_defaults property?